app/presenters/help_page_presenter.rb

```diff
@@ -0,0 +1,2 @@
+class HelpPagePresenter < GovspeakPresenter
```

Comment:
(I think inheritance could go a long way to cleaning up some of presenters, but I'd like to consider more formats when investigation that)